gorm_bulk.go

```diff
@@ -139,7 +130,14 @@ func scopeFromObjects(db *gorm.DB, objects []interface{}, execFunc ExecFunc) (*g
 		}
 
 		for _, key := range columnNames {
-			objectScope.AddToVars(row[key])
+			// Column CreatedAt and UpdatedAt with zero value will be set to same time
+			if name := row[key].Struct.Name; name == "CreatedAt" || name == "UpdatedAt" {
+				if t, ok := row[key].Field.Interface().(time.Time); ok && t.IsZero() {
```

Comment:
I might be wrong but I'm pretty sure `CreatedAt` and `UpdatedAt` can be pointers (even though they're not in the `gorm.Model` type). If so, this type casting can fail since the value might be nil and not a zero time. If that's the case we still want to set the default bulk time, don't we?

Reply:
Yeah, I didn't consider well about the type of `CreatedAt` and `UpdatedAt`. Let's use the old way as `row[key].IsBlank` to check whether the value is set or not.

Reply:
I also changed the statement from `if` to `switch` for better readability.